aspects/aspects.go

```diff
@@ -93,14 +110,13 @@ type Bundle struct {
 
 // NewAspectBundle returns a new aspect bundle for the following aspects
```

Comment:
"for the following" perhaps "for the specified"?

Reply:
:+1: for the first part but, the regarding the second part, access patterns and schemas are different things (in this context). I'll adjust this, thanks